config.example/group_vars/slurm-cluster.yml

```diff
@@ -59,3 +59,10 @@ httpd_listen_addr_port:
 httpd_use_rewrites: false
 node_uri: /node
 rnode_uri: /rnode
+
+################################################################################
+# Set GPU Clocks Adjustment Permission                                                                #
+################################################################################
+set_gpu_clocks: no
```

Comment:
I think a more descriptive variable would be helpful, I thought this had something to do with actually setting the clock speed. Maybe `allow_user_set_gpu_clocks`?

Reply:
Yes I agree, we can change it to that.